UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/SettingsOverrides.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
```

Comment:
Is `SimulationSettingsOverrides` more appropriate?

Reply:
Sure

Reply:
This is directly modifying what unity calls `ProjectSettings`, maybe `ProjectSettingsOverrides`?

Reply:
That works for me too. Since this will only be used in our scenes, I'm less worried about the naming or making sure that it handles all possible cases.